support-frontend/assets/pages/weekly-subscription-checkout/components/thankYou.tsx

```diff
@@ -173,25 +242,66 @@ function ThankYouContent({
 				<HeroImage orderIsGift={orderIsGift} />
 				<HeadingBlock
 					overheadingClass="--thankyou"
-					overheading="Thank you for supporting our journalism!"
+					overheading={thankyouSupportHeader}
 				>
-					{getHeading(billingPeriod, isPending, orderIsGift)}
+					{getHeading(
+						billingPeriod,
+						isPending,
+						orderIsGift,
+						inThreeTierTestVariant,
+					)}
 				</HeadingBlock>
 			</HeroWrapper>
-			<Content>
-				{isPending && (
-					<Text>
-						<LargeParagraph>
-							Your subscription is being processed and you will receive an email
-							when it goes live.
-						</LargeParagraph>
+
+			{inThreeTierTestVariant ? (
+				<>
```

Comment:
Not sure if we need the `Fragment`

Reply:
You do because there are several children – 3 `<Content />`

Reply:
Thanks Max